src/pull_request_comment.ts

```diff
@@ -0,0 +1,37 @@
+import { info, warning } from "@actions/core";
+import { loadContext } from "./context";
+import config from "./config";
+import { initOctokit } from "./octokit";
+import { COMMENT_SIGNATURE } from "./messages";
+
+export async function handlePullRequestComment() {
+  const context = await loadContext();
+  if (context.eventName !== "pull_request_review_comment") {
+    warning("unsupported github event");
+    return;
+  }
+
+  const { comment, pull_request } = context.payload;
+  if (!comment) {
+    warning("`comment` is missing from payload");
+    return;
+  }
+  if (context.payload.action !== "created") {
+    warning("only consider newly created comments");
+    return;
+  }
+  if (!pull_request) {
+    warning("`pull_request` is missing from payload");
+    return;
+  }
+  if (isOwnComment(comment.body)) {
+    info("ignoring own comments");
+    return;
+  }
```

Comment:
@presubmitai can we somehow improve this part?

Reply:
Actually nevermind. It's fine. No longer needed

Reply:
actually you know what, you can actually help me. Can you replace warning statement with console.error?

Reply:
@bstanga Sure! You'll need to update the import statement and replace all instances of `warning()` with `console.error()`. Here's what needs to be changed:

1. Remove `warning` from the import statement on line 1
2. Replace these lines:
```typescript
warning("unsupported github event");
warning("`comment` is missing from payload");
warning("only consider newly created comments");
warning("`pull_request` is missing from payload");
```
with:
```typescript
console.error("unsupported github event");
console.error("`comment` is missing from payload");
console.error("only consider newly created comments");
console.error("`pull_request` is missing from payload");
```